deployment/TODO-next-update.md

```diff
@@ -0,0 +1,79 @@
+# Todo For Next Update
```

Comment:
This is not the newest one, because you didn't used and merged my release `v2.4.0` PR on Ocelot Rebranding Deployment:

https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding/issues/98

Reply:
Fixed in https://github.com/Ocelot-Social-Community/Ocelot-Social/pull/5975/commits/7600d0166579ec4a56b3fad24d239b626efcbfbb